packages/aws-cdk-lib/aws-eks/lib/managed-nodegroup.ts

```diff
@@ -442,6 +446,11 @@ export class Nodegroup extends Resource implements INodegroup {
       }
     }
 
+    // custom AMI type can be used only when there's a launch template that picks an AMI
+    if (props.amiType === NodegroupAmiType.CUSTOM && !props.launchTemplateSpec) {
```

Comment:
Can you please update the comments [here](https://github.com/aws/aws-cdk/blob/da2ec75f6bc8f2e28b07cdf1307b22c83bb652d5/packages/aws-cdk-lib/aws-eks/lib/managed-nodegroup.ts#L196-L197) and [here](https://github.com/aws/aws-cdk/blob/da2ec75f6bc8f2e28b07cdf1307b22c83bb652d5/packages/aws-cdk-lib/aws-eks/lib/managed-nodegroup.ts#L295) accordingly?

Reply:
Added a little bit of clarification in both places.